ui/app/styles/components/masked-input.scss

```diff
@@ -1,13 +1,12 @@
+.masked-font {
+  color: $ui-gray-300;
```

Comment:
I have the masked-font defined in the form.scss however the color is overridden when it cascades to this level.